kiva/agg/setup.py

```diff
@@ -139,11 +139,9 @@ def get_ft2_sources(name_info, build_dir):
     if sys.platform == 'darwin':
         define_macros = [('__DARWIN__', None)]
         macros = [('__DARWIN__', None)]
-        extra_link_args = ['-framework', 'Carbon']
```

Comment:
Dropped adding `framework -Carbon` above since it appears to be unnecessary (compile and link works without it), otherwise the change in how we define the list results in different build error:

```
54 warnings generated.
g++ -bundle -undefined dynamic_lookup -L/Users/jason/miniconda3/envs/py37/lib -arch x86_64 -L/Users/jason/miniconda3/envs/py37/lib -arch x86_64 -arch x86_64 build/temp.macosx-10.7-x86_64-3.7/build/src.macosx-10.7-x86_64-3.7/kiva/agg/agg_wrap.o -Lbuild/temp.macosx-10.7-x86_64-3.7 -lkiva_src -lagg24_src -lfreetype2_src -o build/lib.macosx-10.7-x86_64-3.7/kiva/agg/_agg.cpython-37m-darwin.so -framework ApplicationServices -framework OpenGL Carbon
clang: error: no such file or directory: 'Carbon'
error: Command "g++ -bundle -undefined dynamic_lookup -L/Users/jason/miniconda3/envs/py37/lib -arch x86_64 -L/Users/jason/miniconda3/envs/py37/lib -arch x86_64 -arch x86_64 build/temp.macosx-10.7-x86_64-3.7/build/src.macosx-10.7-x86_64-3.7/kiva/agg/agg_wrap.o -Lbuild/temp.macosx-10.7-x86_64-3.7 -lkiva_src -lagg24_src -lfreetype2_src -o build/lib.macosx-10.7-x86_64-3.7/kiva/agg/_agg.cpython-37m-darwin.so -framework ApplicationServices -framework OpenGL Carbon" failed with exit status 1
```